mods/beds/init.lua

```diff
@@ -3,9 +3,19 @@ beds.player = {}
 beds.pos = {}
 beds.spawn = {}
 
+-- Intllib
+ibeds = {}
```

Comment:
Why do you create an `ibeds` table and then only store the translator in it?  You might as well just make a global `beds_gettext`.  Why not just use the `beds` table though? 

(same applies to boats, bones, bucket, etc)